### DISCUSSION TITLE
eclipse not downloading plugin Tycho Project Configurators	0.9.0.201811261502

### DISCUSSION THREAD

**Discussion by miltonchirinos44** [dbeaver/dbeaver#18126] [DBeaver Development]
Hello friends
due to java update from 11 to 17 by dbeaver decided to re-import the project from the github repository.

when I import the dbeaver project from eclipse it won't let me install `Tycho Project Configurators` and I get the following error window

![image](https://user-images.githubusercontent.com/52106718/200057847-5f0be79a-5d87-4c40-a7cf-f1d93053c1dd.png)

No matter how many times I click the `Next` button, it won't let me install `Tycho Project Configurators`
![image](https://user-images.githubusercontent.com/52106718/200057920-b8e48e81-b046-43ce-aab3-7157c1801b04.png)

I am using the following tools:

- eclipse java Eclipse IDE 2022‑09
- java 17
- maven 3.8.6

**Comment by miltonchirinos44** [comment]
Thanks for the help and I managed to solve the error

**Reply by arons** [reply]
How did you solve the problem? I'm experiencing the same problem.
Thx
